lib/database_cleaner/safeguard.rb

```diff
@@ -52,7 +50,13 @@ def given?
         end
 
         def remote?(url)
-          url && !LOCAL.any? { |str| url.include?(str) }
+          url &&
+            case URI(url).host
+            when nil, 'localhost', /\.local/, '127.0.0.1'
```

Comment:
@faucct Could you keep this as a constant? 

Also, it seems that you missed `sqlite3:` -- was that on purpose?

Reply:
ping